### DISCUSSION TITLE
Lyrics: showing and searching

### DISCUSSION THREAD

**Discussion by bkh85570** [TT-ReBORN/Georgia-ReBORN#187] [General]
My library has on most of the songs synced lyrics. They were saved in the same directory as the song. 
When a song starts the lyrics will be found and displayed. After a few seconds, maybe 30, the displayed lyrics changes to "Search complete No Lyrics were found". Crazy

What have i done wrong in the config?
What information do you need to help me?

**Comment by TT-ReBORN** [comment]
Hi @bkh85570,

if you want to load the lyrics from the same directory as the album, you need to set it up in your config file and ESLyirc component.
I don't know what Georgia-ReBORN version you are using, but it would be the best to download the latest DEV version from here:
https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip

Please follow this instructions from here:
https://github.com/TT-ReBORN/Georgia-ReBORN/issues/175#issuecomment-2017001652

Everything you need to know about custom lyric directory usage is explained in that thread.

-TT

**Comment by bkh85570** [comment]
Thanks for your quick reply. 
Formerly i used the "Georgia-ReBORN_v3.0-RC2.zip" and pasted the files with the new.
I think the main thing, was the new line in in the "lyricsFilenamePatterns" section:
Line 785: "%filename%".

Now it works well.

THX

**Comment by bkh85570** [comment]
After a view months a small question:

are there differences between the master.zip over here and  this on in

https://hydrogenaud.io/index.php/topic,121047.msg1051682.html#msg1051682

i think so, because the searching shows the same handling as before i changed ist

THX bkh85570

**Reply by TT-ReBORN** [reply]
There is no difference, you just need to apply the custom directory as you did before.
When you downloaded the master.zip and replaced ALL your files, you need to re-apply the custom directory in your settings again.
Because it will also replace your config files, i.e your settings will be overwritten with the default ones.

**To not always re-apply this, you first set it up and AFTER configuration make a new theme backup.**
When making a new theme backup, it will save your current settings including those of the config files.
Then when downloading the master.zip ( DEV version ) and replacing your files, restore the new theme backup.

Here is how to do it:

01. Make a theme backup via `Options > Settings > Theme backup > Make backup`.
02. Now create a new portable foobar install and install the latest DEV master version from here:
https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip
03. After you have downloaded and installed the theme, copy from your old foobar your theme backup folder:
`foobar2000_old\profile\backup`
to your new:
`foobar2000\profile\backup`
04. Now restore your theme backup via `Options > Settings > Theme backup > Restore backup`

-TT
